src/apis/index.js

```diff
@@ -0,0 +1,14 @@
+import FireStore from "../firebase/fireStore";
```

Comment:
@GeekySankar you can remove this code if you are using it in some other component